desc/objectives/_qs.py

```diff
@@ -253,25 +280,37 @@ class QuasisymmetryTwoTerm(_Objective):
 
     _scalar = False
     _linear = False
+    _units = "(T^3)"
+    _print_value_fmt = "Quasi-symmetry two-term error: {:10.3e} "
 
     def __init__(
         self,
         eq=None,
         target=0,
         weight=1,
+        normalize=True,
+        normalize_target=True,
         grid=None,
         helicity=(1, 0),
         name="QS two-term",
     ):
 
         self.grid = grid
         self.helicity = helicity
-        super().__init__(eq=eq, target=target, weight=weight, name=name)
-        units = "(T^3)"
+        super().__init__(
+            eq=eq,
+            target=target,
+            weight=weight,
+            normalize=normalize,
+            normalize_target=normalize_target,
+            name=name,
+        )
+
         self._print_value_fmt = (
-            "Quasi-symmetry ({},{}) error: ".format(self.helicity[0], self.helicity[1])
+            "Quasi-symmetry two-term ({},{}) error: ".format(
```

Comment:
Should be `"Quasi-symmetry ({},{}) two-term error: "` if we want to follow the same formatting as the Boozer objective.

Reply:
fixed